### ISSUE TITLE
Add license information

### ISSUE THREAD

**Issue by vazhnov** [microbotsio/MotorCell#1]
Hello!

I found this project from https://hackaday.com/2024/12/17/pcb-motor-holds-fast-even-after-1-6-billion-spins/

Could you please add information about the license of this repository — is it allowed to copy, change and use the code and PCB hardware files in any purpose?
It would be great to see any popular open source license.

**Comment by xegim** [comment]
Seconded. It is currently unknown what are the restrictions regarding its use.

**Comment by vazhnov** [comment]
@xegim , then maybe you can start with more restricted license (like GPLv3) and later shift to less restricted (like MIT) if needed? 🙂

**Comment by xegim** [comment]
Yeah, any open source license would do more or less, but it should be explicitly presented.

**Comment by vazhnov** [comment]
@xegim , sorry, I thought you're one of contributors :)

**Comment by xegim** [comment]
Haha, no worries 😄. Doesn't look like the actual maintainer reads the issues, though...

**Comment by microbotsio** [comment]
Hi! Apologies for the delayed response - we missed the first notification.
The license information has now been updated 🙂
Let us know if you have any other questions!

**Comment by vazhnov** [comment]
Apache-2.0 license — amazing! Thank you!
